prac_06/guitar.py

```diff
@@ -0,0 +1,23 @@
+"""CP1404/CP5632 Practical - Programming Language Do From Scratch Exercise"""
+
+class Guitar:
+        def __init__(self, name, year, cost):
+            self.name = name
+            self.year = year
+            self.cost = cost
+
+        def __str__(self):
+            return "{} ({}) : ${:,.2f}".format(self.name, self.year, self.cost)
+
+        def get_age(self):
+            age = 2018 - self.year
```

Comment:
return Guitar.CURRENT_YEAR - self.year
Based from Lecture recording (Drink and drink list) demo. It suggests to use constants when handling variables that 
won't change.